repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -115,23 +127,31 @@ class ExecutorClassLoader(
     }
   }
 
+  // See org.apache.spark.network.server.TransportRequestHandler.processStreamRequest.
+  private val STREAM_NOT_FOUND_REGEX = s"Stream '.*' was not found.".r.pattern
```

Comment:
Since this exists in `TransportRequestHandler` + here as well and the test mocked it I think nobody will ever realize the problem if the message in `TransportRequestHandler` changed (only on cluster).

Reply:
`nonexistent class and transient errors should cause different errors` doesn't mock `TransportRequestHandler`. It would fail if someone changed the error message.



> I think nobody will ever realize the problem if the message in `TransportRequestHandler` changed (only on cluster).

Good point. I added a comment near the error message so that people can notice the error message string is used here.